Sources/SwiftPlot/PlotStyleHelpers.swift

```diff
@@ -67,3 +67,51 @@ struct Text : Annotation {
         self.location = location
     }
 }
+
+struct Arrow : Annotation {
+    public var color = Color.black
+    public var start = Point(0.0, 0.0)
+    public var end = Point(0.0, 0.0)
+    public var strokeWidth: Float = 10
+    public var headLength: Float = 10
+    public var headAngle: Float = 20
+    public func draw(renderer: Renderer) {
+        // Draws arrow body.
+
+        //TODO: scale points to be plotted according to plot size
+        
+        renderer.drawLine(startPoint: start,
+                          endPoint: end,
+                          strokeWidth: strokeWidth,
+                          strokeColor: color,
+                          isDashed: false)
+
+        // Calculates arrow head points.
+        var p1 = end + Point(cos(headAngle)*headLength, sin(headAngle)*headLength)
+        var p2 = end + Point(cos(headAngle)*headLength, -sin(headAngle)*headLength)
+
+        let rotateAngle = -atan2(start.x - end.x, start.y - end.y)
+        p1 = rotatePoint(point: p1, center: end, angleRadians: rotateAngle + 0.5 * Float.pi)
+        p2 = rotatePoint(point: p2, center: end, angleRadians: rotateAngle + 0.5 * Float.pi)
+
+        // Draws arrow head points.
+        renderer.drawLine(startPoint: end,
+                          endPoint: p1,
+                          strokeWidth: strokeWidth,
+                          strokeColor: color,
+                          isDashed: false)
+        renderer.drawLine(startPoint: end,
+                          endPoint: p2,
+                          strokeWidth: strokeWidth,
+                          strokeColor: color,
+                          isDashed: false)
```

Comment:
Here you can use `renderer.drawPlotLines` and pass it an array of the points so that for higher thicknesses it doesn't get like this:
<img width="157" alt="Screenshot 2019-12-23 at 12 47 46" src="https://user-images.githubusercontent.com/278662/71358880-6e9d4c00-2582-11ea-8f4f-af721e8d7038.png">